services/ui/ws/platform_display.cc

```diff
@@ -34,5 +34,11 @@ std::unique_ptr<PlatformDisplay> PlatformDisplay::Create(
 #endif
 }
 
+bool PlatformDisplay::RunMoveLoop(const gfx::Vector2d& drag_offset) {
```

Comment:
How do the events flow? In particular, why can't `mus` short-circuit the event flow and listen to the incoming event stream to position the window? Is this is a dumb idea?

Reply:
Well, I've tried that approach, but the movement of the window wasn't smooth enough. It was jumping back and forth while dragging.